### PR TITLE
feat: Improve Terraform

### PR DESCRIPTION
**Pull request by HarryET**
# Description
Terraform now gets the default version if not provided

## How Has This Been Tested?
N/a

## Due Diligence

* [ ] Breaking change
* [ ] Requires a documentation update
* [ ] Requires a e2e/integration test update

### CODE REVIEW

**Review by arein — APPROVED** [review]

**Comment by HarryET** [comment]
Waiting on Bat Cave to test these changes work before merging. Will merge once verified!

**Pull request merged**
